client/js/components/Container.jsx

```diff
@@ -17,13 +17,19 @@ export default class Container extends React.Component {
     }
 
     render() {
+        // Pass props to dynamically rendered children
+        var props = { slideUp: this.slideUp.bind(this) };
+        const children = React.Children.map(this.props.children, (element) =>
+            React.cloneElement(element, props)
+        );
```

Comment:
Would it be cleaner to use `...` to transfer the props? https://facebook.github.io/react/docs/transferring-props.html